beetle-core/src/main/java/com/xing/beetle/amqp/RequeueAtEndConnection.java

```diff
@@ -120,6 +120,7 @@ public void basicReject(long deliveryTag, boolean requeue) throws IOException {
           delegate.basicReject(deliveryTag, false);
         } else {
           // silently drop the message by accepting
+          log.warn("message with delivery tag silently dropped " + deliveryTag);
```

Comment:
If the drop is supposed to be silent, shouldn't this be a debug level log rather?

Reply:
Yes, I agree.